megamek/src/megamek/common/autoresolve/acar/SimulationContext.java

```diff
@@ -428,8 +442,13 @@ public List<Formation> getActiveDeployedFormations() {
     }
 
     public List<Formation> getActiveFormations(Player player) {
+        return this.getActiveFormations(player.getId());
```

Comment:
Can player ever be null?

Reply:
Not that I am aware of. I will do an analysis of it, prob its safe and if it ever receive a null is a bug and has to be fixed

Reply:
No, in ACAR its not possible to have a player disappear from a game, and the way the data is accessed it has to be an error if there is no player because its an unrecoverable situation.